api/swagger/swagger.yaml

```diff
@@ -147,6 +147,35 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/ErrorResponse"
+  /event/{lng},{lat},{radius}:
```

Comment:
it doesn't have to be a new endpoint. (also should be `/events/`, plural) So `/events/` returns a list of event object,  and supports pagination, sorted by distance by default, but it should be able to take a `sort` parameter, possible value: `created_at`.
````
var options = {}
if (lat && lng) {
    options.$near = [lat, lng];
    options.distance = radius || defaultRadius (let's say 5km)
}

if (sort === "created_at") {
    options.sort = {createdAt: -1} 
}

Event.find(opionts).skip(page-1).limit(pageSize)
......
```